OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -84,6 +88,13 @@ void CaptureClient::Capture(int32_t process_id, std::string process_name,
     instrumented_function->set_function_type(IntrumentedFunctionTypeFromOrbitType(function.type()));
   }
 
+  for (const auto& tracepoint : selected_tracepoints) {
+    CaptureOptions::InstrumentedTracepoint* instrumented_tracepoint =
+        capture_options->add_instrumented_tracepoint();
```

Comment:
Can you reuse TracepontInfo proto here? it looks pretty much the same.

Reply:
I was actually curious how I could use TracepointInfo. Thank you so much for the quick chat in which you have explained me!